etc/docker/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.3"
+
+services:
+
+  enterprise-gateway:
+    image: elyra/enterprise-gateway:${VERSION:-2.0.0rc1}
+    user: root
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock"
```

Comment:
Would it be possible to add the `MOUNT_KERNELSPECS` entry but commented out?

Reply:
Yes, I've added this and the documentation.